src/pages/gen2/build-a-backend/add-aws-services/deletion-backup-resources/index.mdx

```diff
@@ -0,0 +1,110 @@
+export const meta = {
+  title: 'Deletion protection and Backup resources',
+  description: 'Learn how to enable deletion protection and backup on resources'
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      meta
+    }
+  };
+}
+
+
+<Callout warning>
+
+Deleting a Amplify sandbox with a resource enabled with deletion protection, the deploy process will fail and the resource will need to be manually deleted on the AWS console. 
+
+</Callout>
+
+Using AWS CDK we can configure Amplify generated resource to enable deletion protection and backups on supported resources. For example, a developer can could use CDK to enable Point-in-time recovery for DynamoDB tables, or use AWS Backups as a advanced backup option.
```

Comment:
```suggestion
Using the [AWS Cloud Development Kit (CDK)](https://docs.aws.amazon.com/cdk/latest/guide/home.html) we can configure Amplify generated resource to enable deletion protection and backups on supported resources. For example, you can use CDK to enable [Point-in-time recovery for DynamoDB tables](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/PointInTimeRecovery.html), or use [AWS Backup](https://aws.amazon.com/backup/) as a advanced backup option.
```

first mention of CDK should include the full name and acronym. also added a few docs links for the features. 

"a developer can" -> "you can"